sqlglot/generator.py

```diff
@@ -2271,11 +2271,14 @@ def altertable_sql(self, expression: exp.AlterTable) -> str:
         actions = expression.args["actions"]
 
         if isinstance(actions[0], exp.ColumnDef):
-            actions = self.expressions(
-                expression,
-                key="actions",
-                prefix="ADD COLUMN " if self.ALTER_TABLE_ADD_COLUMN_KEYWORD else "ADD ",
-            )
+            if self.ALTER_TABLE_ADD_COLUMN_KEYWORD:
+                actions = self.expressions(
+                    expression,
+                    key="actions",
+                    prefix="ADD COLUMN ",
+                )
+            else:
+                actions = f"ADD {self.expressions(expression,key='actions')}"
```

Comment:
you're missing a space? did you run make style?

Reply:
why isn't this self.expressions(prefix...) like the previous?

Reply:
I ran `pre-commit run` on the staged files before making the commit. It reformatted the tests, didn't do anything here. Fixed now.